### PR TITLE
Mention explicitly that  unsignedInteger must not be larger than Long.MAX_VALUE.

### PR DESCRIPTION
**Pull request by Lojjs**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-841.surge.sh

Updated pages:
- [Syntax and semantics](https://neo4j-docs-cypher-841.surge.sh/cypher-manual/5/patterns/reference)

**Review by loveleif — APPROVED** [review]
Sorry forgot about this one!

File: modules/ROOT/pages/patterns/reference.adoc [hunk 1]
```diff
@@ -760,6 +760,11 @@ upperBound ::= unsignedInteger
 unsignedInteger ::= [0-9]+
 ----
 
+[NOTE]
+====
+The `unsignedInteger` must not be larger than the Java constant `Long.MAX_VALUE`.
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/patterns/reference.adoc]: Maybe this is not necessary, but helpful if readers are both unaware and interested in what the Long.MAX_VALUE is? If you agree - same comment goes for below note. 

```suggestion
The `unsignedInteger` must not be larger than the Java constant `Long.MAX_VALUE` (2^63-1).
```

**Review by JPryce-Aklundh — APPROVED** [review]

File: modules/ROOT/pages/patterns/reference.adoc (hunk 1 above)
Comment (Lojjs) [reply modules/ROOT/pages/patterns/reference.adoc]: Good idea, added.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-841.surge.sh

Updated pages:
- [Syntax and semantics](https://neo4j-docs-cypher-841.surge.sh/cypher-manual/5/patterns/reference)

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Pull request merged**